Quarkus test fails on Jenkins

My application runs tests very well on my local system. However if I want to execute mvn test on my Jenkins pipeline I encounter problems that might be related to the proxy. I set my proxy for all mvn commands and I see that the dependencies are fetched. During the test phase it breaks with the following error. Is there any dependency loading outside the maven context? Is 

`
[INFO] 
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running backend.FeedbackControllerTest
[ERROR] Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 4.164 s <<< FAILURE! - in backend.FeedbackControllerTest
[ERROR] testEmptyProjectList  Time elapsed: 0.022 s  <<< ERROR!
org.junit.jupiter.api.extension.TestInstantiationException: TestInstanceFactory [io.quarkus.test.junit.QuarkusTestExtension] failed to instantiate test class [backend.FeedbackControllerTest]: Failed to create the boostrap class loader
Caused by: java.lang.IllegalStateException: Failed to create the boostrap class loader
Caused by: io.quarkus.bootstrap.BootstrapException: Failed to create the deployment classloader for backend::jar:1.0-SNAPSHOT
Caused by: io.quarkus.bootstrap.resolver.AppModelResolverException: Failed to inject extension deployment dependencies for backend:jar:1.0-SNAPSHOT
Caused by: io.quarkus.bootstrap.resolver.maven.DeploymentInjectionException: io.quarkus.bootstrap.resolver.AppModelResolverException: Failed to collect dependencies for io.quarkus:quarkus-resteasy-deployment:jar:0.19.1
Caused by: io.quarkus.bootstrap.resolver.AppModelResolverException: Failed to collect dependencies for io.quarkus:quarkus-resteasy-deployment:jar:0.19.1
Caused by: org.eclipse.aether.collection.DependencyCollectionException: Failed to read artifact descriptor for io.quarkus:quarkus-resteasy-deployment:jar:0.19.1
Caused by: org.eclipse.aether.resolution.ArtifactDescriptorException: Failed to read artifact descriptor for io.quarkus:quarkus-resteasy-deployment:jar:0.19.1
Caused by: org.eclipse.aether.resolution.ArtifactResolutionException: Could not transfer artifact io.quarkus:quarkus-resteasy-deployment:pom:0.19.1 from/to central (https://repo.maven.apache.org/maven2): repo.maven.apache.org: Name or service not known
Caused by: org.eclipse.aether.transfer.ArtifactTransferException: Could not transfer artifact io.quarkus:quarkus-resteasy-deployment:pom:0.19.1 from/to central (https://repo.maven.apache.org/maven2): repo.maven.apache.org: Name or service not known
Caused by: java.net.UnknownHostException: repo.maven.apache.org: Name or service not known
`

--- a/devtools/maven/pom.xml
+++ b/devtools/maven/pom.xml
@@ -21,6 +21,12 @@
         <dependency>
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-bootstrap-core</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.maven.wagon</groupId>
+                    <artifactId>wagon-provider-api</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>io.quarkus</groupId>

--- a/independent-projects/bootstrap/core/pom.xml
+++ b/independent-projects/bootstrap/core/pom.xml
@@ -32,13 +32,17 @@
         </dependency>
         <dependency>
             <groupId>org.apache.maven.resolver</groupId>
-            <artifactId>maven-resolver-transport-file</artifactId>
+            <artifactId>maven-resolver-transport-wagon</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.apache.maven.resolver</groupId>
-            <artifactId>maven-resolver-transport-http</artifactId>
+            <groupId>org.apache.maven.wagon</groupId>
+            <artifactId>wagon-http</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.apache.maven.wagon</groupId>
+            <artifactId>wagon-file</artifactId>
+        </dependency>
+         <dependency>
             <groupId>org.jboss.logging</groupId>
             <artifactId>jboss-logging</artifactId>
         </dependency>

--- a/independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/resolver/maven/BootstrapWagonProvider.java
+++ b/independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/resolver/maven/BootstrapWagonProvider.java
@@ -1,0 +1,84 @@
+package io.quarkus.bootstrap.resolver.maven;
+
+import org.apache.maven.wagon.Wagon;
+import org.eclipse.aether.transport.wagon.WagonProvider;
+
+public class BootstrapWagonProvider implements WagonProvider {
+
+    @Override
+    public Wagon lookup(String roleHint) throws Exception {
+        String impl = null;
+        switch(roleHint) {
+            case "https":
+            case "http":
+                impl = "org.apache.maven.wagon.providers.http.HttpWagon";
+                break;
+            case "file":
+                impl = "org.apache.maven.wagon.providers.file.FileWagon";
+                break;
+            case "ftp":
+                impl = "org.apache.maven.wagon.providers.ftp.FtpWagon";
+                break;
+            case "ftps":
+                impl = "org.apache.maven.wagon.providers.ftp.FtpsWagon";
+                break;
+            case "ftph":
+                impl = "org.apache.maven.wagon.providers.ftp.FtpHttpWagon";
+                break;
+            case "scm":
+                impl = "org.apache.maven.wagon.providers.scm.ScmWagon";
+                break;
+            default:
+                throw new IllegalStateException("Not supported Wagon implementation hint " + roleHint);
+        }
+        final Class<?> cls = loadClass(impl, roleHint);
+        final Object wagon;
+        try {
+            wagon = cls.newInstance();
+        } catch(Throwable t) {
+            throw new IllegalStateException("Failed to instantiate Wagon impl " + impl, t);
+        }
+        return Wagon.class.cast(wagon);
+    }
+
+    @Override
+    public void release(Wagon wagon) {
+    }
+
+    private static Class<?> loadClass(String name, String protocol) {
+        try {
+            return Thread.currentThread().getContextClassLoader().loadClass(name);
+        } catch (ClassNotFoundException e) {
+            final StringBuilder buf = new StringBuilder();
+            buf.append("Failed to locate Wagon implementation ")
+            .append(name)
+            .append(" on the classpath for protocol ")
+            .append(protocol)
+            .append(", please add the ");
+            final String defaultWagonGA = getDefaultWagonGA(protocol);
+            if(defaultWagonGA == null) {
+                buf.append("corresponding classpath dependency to your project");
+            } else {
+                buf.append("desired version of ").append(defaultWagonGA).append(" as a classpath dependency to your project");
+            }
+            throw new IllegalStateException(buf.toString());
+        }
+    }
+
+    private static String getDefaultWagonGA(String protocol) {
+        switch(protocol) {
+            case "https":
+            case "http":
+                return "org.apache.maven.wagon:wagon-http or org.apache.maven.wagon:wagon-http-lightweight";
+            case "file":
+                return "org.apache.maven.wagon:wagon-file";
+            case "ftp":
+            case "ftps":
+            case "ftph":
+                return "org.apache.maven.wagon:wagon-ftp";
+            case "scm":
+                return "org.apache.maven.wagon:wagon-scm";
+        }
+        return null;
+    }
+}

--- a/independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/resolver/maven/MavenRepoInitializer.java
+++ b/independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/resolver/maven/MavenRepoInitializer.java
@@ -55,8 +55,8 @@ import org.eclipse.aether.repository.RemoteRepository;
 import org.eclipse.aether.repository.RepositoryPolicy;
 import org.eclipse.aether.spi.connector.RepositoryConnectorFactory;
 import org.eclipse.aether.spi.connector.transport.TransporterFactory;
-import org.eclipse.aether.transport.file.FileTransporterFactory;
-import org.eclipse.aether.transport.http.HttpTransporterFactory;
+import org.eclipse.aether.transport.wagon.WagonProvider;
+import org.eclipse.aether.transport.wagon.WagonTransporterFactory;
 import org.eclipse.aether.util.repository.AuthenticationBuilder;
 import org.eclipse.aether.util.repository.DefaultAuthenticationSelector;
 import org.eclipse.aether.util.repository.DefaultMirrorSelector;
@@ -152,8 +152,8 @@ public class MavenRepoInitializer {
         final DefaultServiceLocator locator = MavenRepositorySystemUtils.newServiceLocator();
         if(!offline) {
             locator.addService(RepositoryConnectorFactory.class, BasicRepositoryConnectorFactory.class);
-            locator.addService(TransporterFactory.class, FileTransporterFactory.class);
-            locator.addService(TransporterFactory.class, HttpTransporterFactory.class);
+            locator.addService(TransporterFactory.class, WagonTransporterFactory.class);
+            locator.setServices(WagonProvider.class, new BootstrapWagonProvider());
         }
         locator.setServices(ModelBuilder.class, new MavenModelBuilder(wsModelResolver));
         locator.setErrorHandler(new DefaultServiceLocator.ErrorHandler() {

--- a/independent-projects/bootstrap/pom.xml
+++ b/independent-projects/bootstrap/pom.xml
@@ -29,6 +29,7 @@
         <maven-core.version>3.5.4</maven-core.version>
         <maven-plugin-annotations.version>3.5.2</maven-plugin-annotations.version>
         <maven-resolver.version>1.1.1</maven-resolver.version>
+        <maven-wagon.version>3.0.0</maven-wagon.version>
         <nexus-staging-maven-plugin.version>1.6.8</nexus-staging-maven-plugin.version>
     </properties>
     <modules>
@@ -112,13 +113,18 @@
             </dependency>
             <dependency>
                 <groupId>org.apache.maven.resolver</groupId>
-                <artifactId>maven-resolver-transport-file</artifactId>
+                <artifactId>maven-resolver-transport-wagon</artifactId>
                 <version>${maven-resolver.version}</version>
             </dependency>
             <dependency>
-                <groupId>org.apache.maven.resolver</groupId>
-                <artifactId>maven-resolver-transport-http</artifactId>
-                <version>${maven-resolver.version}</version>
+                <groupId>org.apache.maven.wagon</groupId>
+                <artifactId>wagon-file</artifactId>
+                <version>${maven-wagon.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.maven.wagon</groupId>
+                <artifactId>wagon-http</artifactId>
+                <version>${maven-wagon.version}</version>
             </dependency>
             <dependency>
                 <groupId>org.jboss.logging</groupId>

--- a/integration-tests/tika/pom.xml
+++ b/integration-tests/tika/pom.xml
@@ -34,6 +34,16 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-junit5</artifactId>
             <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.jsoup</groupId>
+                    <artifactId>jsoup</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.slf4j</groupId>
+                    <artifactId>jcl-over-slf4j</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>io.rest-assured</groupId>

